db/functions/Makefile

```diff
@@ -1,7 +1,7 @@
 PG_CONFIG ?= pg_config
 DESTDIR ?= .
 
-QTDIR=../../lib/quad_tile
+QTDIR=./
```

Comment:
I was already trying to figure how best to make this find the header in the quad_tile gem ;-)

I mean duplicating it like this is an option but I wonder if it's better to avoid duplication by having the gem export the path to the header?

Reply:
I was wondering about this myself. I can take a spin at how that might look. Ive always been curious what the best way to handle shared dependencies within C code is.

Reply:
Ahh I think I see what your saying now, the Gem could expose this in some way. Ill add that over in the quad_tile repo

Reply:
Turns out bundler has a hand way to do this with `bundle show`. Since we `bundle install` before this step we can assume its installed at this step. 👍

Reply:
Would be nice however if we could figure out a way to extract this to the gem as well (possibly renaming to something like `osm_extensions`) and avoid any manual compiling step in the install. Might be a clever way to do this with the pg gem. Ill look into it.